rna-transcription.json

```diff
@@ -0,0 +1,105 @@
+{
+   "#": [
+     "Language implementations vary on the issue of missing input variables.",
+     "In this set of test cases, the input variable is left out of the definition,",
+     "and the generator would need to handle the language implementation."
```

Comment:
If we use an empty string, will we not run into the scenario where we could really be wanting an empty string returned? Maybe a better way to flag an exception (acknowledge language dependant reference:) ) would be to not have an expected tag, and then the generator could take the absence of a return value as the indicator of an exception?

Basically the return value is undefined, so why have it in the test json?